modules/solid_mechanics/include/materials/SolidModel.h

```diff
@@ -143,6 +143,7 @@ class SolidModel : public DerivativeMaterialInterface<Material>
   MaterialProperty<SymmTensor> & _d_stress_dT;
 
   SymmTensor _total_strain_increment;
+  SymmTensor _mechanical_strain_increment;
   SymmTensor _strain_increment;
```

Comment:
Let's put comments on these three variables to clarify what they are:
/// Total strain increment, including mechanical strains and eigenstrains
SymmTensor _total_strain_increment;
/// Mechanical strain increment, which is the total strain increment minus eigenstrains
SymmTensor _mechanical_strain_increment;
/// In most models, this is the mechanical strain increment, but for inelastic models, it has the inelastic
/// compontent subtracted from it, so it is the elastic strain increment.
SymmTensor _strain_increment;